src/NzbDrone.Core/Music/TrackRepository.cs

```diff
@@ -48,7 +48,7 @@ public Track Find(int artistId, int albumId, int trackNumber)
 
         public List<Track> GetTracks(int artistId)
         {
-            return Query.Join<Track, Artist>(JoinType.Inner, s => s.Artist, (track, artist) => track.ArtistId == artist.Id).ToList();
```

Comment:
We can leave the join and change to left join if you need artist loaded for track parsing. Left should still return tracks if there is no artist link